### PR TITLE
Fixes the broken build - logger

### LINKED ISSUE #1257: make all fails with logger not found
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
The `make all` fails because this fails to run ` go build -o bin/logger ./cmd/logger` . The `cmd` does not have `logger`

**What did you expect to happen:**
`make` to succeed. 

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Istio Version:
- Knative Version:
- KFServing Version:
- Kubeflow version:
- Kfdef:[k8s_istio/istio_dex/gcp_basic_auth/gcp_iap/aws/aws_cognito/ibm]
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

### PR DESCRIPTION
**Pull request by naveensrinivasan**
The make fails with the broken build because the logger is not present in the repository.

**What this PR does / why we need it**:
Fixes the broken build
**Which issue(s) this PR fixes will close the issue(s) when PR gets merged)**:
Fixes #1257

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
NONE

### CODE REVIEW

**Comment by pugangxa** [comment]
Thanks for your PR, would you please also remove other useless target for logger, like docker-build-logger, docker-push-logger in Makefile, and also the dockerfile for logger?

**Comment by naveensrinivasan** [comment]
> Thanks for your PR, would you please also remove other useless target for logger, like docker-build-logger, docker-push-logger in Makefile, and also the dockerfile for logger?

@pugangxa  Took care of it. Thanks

File: logger.Dockerfile [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-
-# Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o agent ./cmd/agent
-
-# Copy the inference-agent into a thin image
-FROM gcr.io/distroless/static:latest
-COPY third_party/ third_party/
-WORKDIR /
-COPY --from=builder /go/src/github.com/kubeflow/kfserving/agent .
-ENTRYPOINT ["/agent"]
```
Comment (yuzisun) [inline logger.Dockerfile:22]: Do not remove this yet, it is still part of the CI although it produces the same image as agent. We will take care of this.

Comment (pugangxa) [reply logger.Dockerfile:22]: Got it. So we can remove it later after CI fixed.

**Comment by naveensrinivasan** [comment]
@PatrickXYS Thanks! 

Looks like the build succeeded!

File: logger.Dockerfile (hunk 1 above)
Comment (PatrickXYS) [reply logger.Dockerfile:22]: @naveensrinivasan 

Seems like you should not remove this as of now

Comment (naveensrinivasan) [reply logger.Dockerfile:22]: Thanks

**Comment by yuzisun** [comment]
Thanks @naveensrinivasan ! We have fixed this, so the PR is no longer needed, thanks for reporting the issue.

**Pull request closed without merging**